builtin/credential/ldap/backend.go

```diff
@@ -11,7 +11,10 @@ import (
 	"github.com/hashicorp/vault/sdk/logical"
 )
 
-const errUserBindFailed = `ldap operation failed: failed to bind as user`
+const (
+	operationPrefixLDAPAuth = "ldap-auth"
```

Comment:
is the `auth` to differentiate between secret engines?

Reply:
Yes, there are some conflicts with the LDAP secrets engine, "Config" for example

Reply:
Switched this to suffix for config